platforms/windows/qlcplus5Qt6.nsi

```diff
@@ -0,0 +1,154 @@
+;--------------------------------
```

Comment:
This file had been added in the commit with unrelated message, about fixing includes in e131controller.h

Reply:
Ouch might have slipped in into a larger commit. My bad